go/registry/api/api.go

```diff
@@ -1005,6 +1031,10 @@ type ConsensusParameters struct {
 	// allowed outside of the genesis block.
 	DebugAllowRuntimeRegistration bool `json:"debug_allow_runtime_registration"`
 
+	// DebugAllowTestRuntimes is true iff test runtimes should be allowed to
+	// be registered.
+	DebugAllowTestRuntimes bool `json:"debug_allow_test_runtimes"`
```

Comment:
Require "don't blame oasis"?

Reply:
I can see external developers using this flag, unlike a lot of the other ones that are feature gated that way.  If we are serious about having third parties writing runtimes, it is probably best not to feature gate it that way?

Reply:
Yeah, makes sense.